manifests/mount.pp

```diff
@@ -9,7 +9,8 @@
   $cvmfs_max_ttl = undef,
   $cvmfs_env_variables = undef,
   $cvmfs_use_geoapi = undef,
-  $cvmfs_repo_list = true
+  $cvmfs_repo_list = true,
+  $cmvfs_mount_rw = 'no',
```

Comment:
I think this should be an undef.

I've just written a test to check the that the default configuration does not change so this should be caught now.
It does not work for the main default.local file but works for the domain and mount.

Apologies for the delay.